src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -229,54 +230,76 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   def adminDeleteWorkspace(workspaceName: WorkspaceName): Future[PerRequestMessage] = asFCAdmin {
     dataSource.inTransaction { dataAccess =>
       withWorkspaceContext(workspaceName, dataAccess) { workspaceContext =>
-        deleteWorkspace(workspaceName, dataAccess, workspaceContext)
+        DBIO.successful(workspaceContext)
```

Comment:
side note: we should keep an eye on this pattern. if we end up doing it a lot, we should refactor to `getWorkspaceContext` that includes the txn inside it